packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -309,11 +318,7 @@ class OnsetPicker extends React.Component {
           }}
           selectId={`${this.props.id}-granularity-select`}
         >
-          <SelectField.Option
-            value={GranularityOptions.AGE}
-            display={intl.formatMessage({ id: 'Terra.onsetPicker.age' })}
-            key={GranularityOptions.AGE}
-          />
```

Comment:
This is probably a question for UX / SD's. Having the option disappear completely based on age might be confusing to users. 

A couple potential alternatives:

1. Disabled the age option if the age falls below 1 week. This preserves the option within the list, but makes it un-selectable. ( This would need UX / SD approval, disabled options are sometimes frowned upon )
2. Allow the user to select an age granularity but display a new message or component if the age falls below 1 week. ( instead of an empty Select )

cc: @neilpfeiffer @PaulSolDev

Reply:
**UX Input**

My design standards discussion ruled out option 1 since our design philosophy on disabled items is that the assumption is that there's something that can be done elsewhere to enable that item. So, since birthdate is fixed, that can't be.

Option 2 adds UI cruft.

Option 3 is the best option. But, also according to design standards, we should contextually only give options that are applicable. So, in this case, until a patient is 1 week old, age should disappear. And, perhaps Year would disappear until the patient is one year old. Month could still be applicable for that age patient, but Date seems like a default choice for a patient less than one week old.

Reply:
However, there's more at play here that could inform other design solutions. [I replied in the original issue](https://github.com/cerner/terra-clinical/issues/535#issuecomment-540799446) in order to get more context. If this is a NICU scenario, it makes sense to have granularity down to hours or even minutes.

Reply:
@jeremyfuksa for the current consumers of this control, hours and minutes aren't supported as a mechanism to pick the age.  They actually get stored in the database as a number and a unit and the only units available are days, weeks, months.  If we have a consumer in the future that supports hours and minutes, I think we could extend the control to allow those additional units and update the logic to drop age if the patient is less than the minimum amount old, but for now I think we just drop the age if they're under a year.